### PR TITLE
DOC: Changed shape assignment example to reshape. Elaborated modifying shape.

### PR DESCRIPTION
**Pull request by petetanru**

### CODE REVIEW

**Comment by petetanru** [comment]
sorry that this is the 3rd PR. i couldn't figure out how to properly squash commits from PR..

File: doc/source/user/quickstart.rst [hunk 1]
```diff
@@ -626,14 +626,28 @@ An array has a shape given by the number of elements along each axis::
     >>> a.shape
     (3, 4)
 
-The shape of an array can be changed with various commands::
+The shape of an array can be changed with various commands. Note that the
+following three commands all return a modified array, but does not change
```
Comment (seberg) [inline doc/source/user/quickstart.rst]: but does -> do not change

**Comment by seberg** [comment]
No worries, usually its `DOC` not `DOCS` but we don't care *that* much ;). If you want to revise a commit, you can use `git commit --amend` (and then forced push). If you want to squash things, from the console `git rebase -i HEAD~5` (for the last 5 commits) is pretty nice. There you can edit things to squash, or edit individual commits.

**Comment by petetanru** [comment]
ok i know my mistake now, when i rebase i left things there as "pick" rather than comment it out, which is why i've been doing a bunch of other people's commit.. 

starting a new PR since i don't know how to rewind from here now

**Pull request closed without merging**